test/e2e/suite/conformance/certificates/acme/acme.go

```diff
@@ -54,15 +65,21 @@ var _ = framework.ConformanceDescribe("Certificates", func() {
 		certificates.KeyUsagesFeature,
 	)
 
-	provisionerHTTP01 := new(acmeIssuerProvisioner)
+	provisionerHTTP01 := &acmeIssuerProvisioner{
+		eab: eab,
+	}
+
+	provisionerDNS01 := &acmeIssuerProvisioner{
+		eab: eab,
+	}
```

Comment:
Does this mean we *only* test with EAB enabled now? I'd be happy to just see the suite run using e.g. HTTP01 with EAB enabled (I don't think we really need to run the full DNS01 && HTTP01 suite with EAB enabled), but I'd also like to see this run *without* EAB enabled too (as before).

Reply:
This should run the acme tests with and without EAB. the passed in eab struct can be nil